modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js

```diff
@@ -410,6 +439,7 @@ class CriteriaRow extends Component {
 				{error ? (
 					<ClayButton
 						className="btn-outline-danger btn-sm"
+						displayType=""
```

Comment:
🤔

Reply:
this button can be improve. Trouble is: `displayType`defaults to `primary` and overwrites `btn-outline-danger`. I'm sad

Reply:
🙁